src/Context/liked-videos-context.js

```diff
@@ -0,0 +1,31 @@
+import { useReducer, createContext, useContext } from "react"
+
+const LikedVideosContext = createContext()
+
+const updateLikedVideosFunc = (state,action) => {
+    switch(action.type)
+    {
+        case "UPDATE_LIKED_VIDEOS_LIST" : 
+            {
+                return [...action.payload]
+            }
+        default :
+            {
+                return [...state]
```

Comment:
this could have been acceptable as `return state` also

Reply:
It is just to make sure that reducer is pure function and app re-renders even on default case